src/UI/SignTransactionsModals/SignWithDeviceModal/components/components/ConfirmReceiver/ConfirmReceiver.tsx

```diff
@@ -1,34 +1,100 @@
 import React from 'react';
-
 import { faExclamationTriangle } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import classNames from 'classnames';
+
+import { ACCOUNTS_ENDPOINT } from 'apiCalls';
+import MultiversXIconSimple from 'assets/icons/mvx-icon-simple.svg';
+import { useGetAccountFromApi } from 'hooks';
+import { CopyButton } from 'UI/CopyButton';
+import { ExplorerLink } from 'UI/ExplorerLink';
+import { LoadingDots } from 'UI/LoadingDots';
+import { Trim } from 'UI/Trim';
+import { isContract } from 'utils/smartContracts';
+
 import styles from './confirmReceiverStyles.scss';
+import { trimReceiverDomain } from './helpers';
 
 export interface ConfirmReceiverPropsType {
   receiver?: string;
   scamReport: string | null;
 }
 
-export const ConfirmReceiver = (props: ConfirmReceiverPropsType) => {
-  const { receiver, scamReport } = props;
+export const ConfirmReceiver = ({
+  receiver,
+  scamReport
+}: ConfirmReceiverPropsType) => {
+  const isSmartContract = isContract(receiver ?? '');
```

Comment:
Does it make sense to proceed with the code if receiver is missing?

Reply:
Updated. 👍